
The usual tweaking for 2.6 haeder files.

  - Rene <rene@exactcode.de>

--- ./backlight.c.orig	2004-10-14 14:28:03.000000000 +0200
+++ ./backlight.c	2004-10-14 14:28:11.000000000 +0200
@@ -24,7 +24,6 @@
 #include <sys/types.h>
 #include <linux/ioctl.h>
 #include <linux/types.h>
-#include <linux/adb_mouse.h>
 #include <linux/cuda.h>
 #include <linux/adb.h>
 #include <linux/pmu.h>
--- ./trackpad.c.orig	2004-10-14 14:27:39.000000000 +0200
+++ ./trackpad.c	2004-10-14 14:27:46.000000000 +0200
@@ -18,7 +18,6 @@
 #include <unistd.h>
 #include <errno.h>
 #include <fcntl.h>
-#include <linux/adb_mouse.h>
 #include <linux/cuda.h>
 #include <linux/adb.h>
 
--- ./vmode.c.orig	2004-10-14 14:35:18.000000000 +0200
+++ ./vmode.c	2004-10-14 14:40:58.000000000 +0200
@@ -4,7 +4,22 @@
 #include <sys/types.h>
 #include <sys/ioctl.h>
 #include <sys/vt.h>
-#include <asm/vc_ioctl.h>
+
+#define VC_GETMODE 0x7667
+#define VC_SETMODE 0x7668
+
+struct vc_mode {
+        int     height;
+        int     width;
+        int     depth;
+        int     pitch;
+        int     mode;
+        char    name[32];
+        unsigned long fb_address;
+        unsigned long cmap_adr_address;
+        unsigned long cmap_data_address;
+        unsigned long disp_reg_address;
+};
 
 int die(char *str)
 {
